runtime/caml/signals.h

```diff
@@ -30,6 +30,15 @@
 extern "C" {
 #endif
 
+CAMLextern void caml_enter_blocking_section (void);
+CAMLextern void caml_leave_blocking_section (void);
+
+CAMLextern void caml_process_pending_actions (void);
+/* Checks for pending actions and executes them. This includes pending
+   minor and major collections, signal handlers, finalisers, and
+   Memprof callbacks. Assumes that the runtime lock is held. Can raise
+   exceptions asynchronously into OCaml code. */
```

Comment:
I welcome style comments in general. Once again, the style in the codebase is inconsistent (see for instance `caml.weak.h` which does the contrary and uses `/** */` comments). Is there are use for `/** */` currently?

Reply:
I think `caml:weak.h` is the (almost ?) only file that does this. And no, there is no automatic documentation generated from `.h` files, so `/** */` are of no use.

Reply:
It still makes a good proportion of those that are nicely documented. In fact there is not a single consistent style:
backtrace, osdeps: before
intext: after, indented
memory: before, additional whitespace
(And I do care, because if there is a consistent style it means that I do not have to spend time thinking about it. Being asked by the contribution guidelines to copy the surrounding style when there is no consistency is the worst.)

Reply:
Alright. Do as you prefer.